docs/PUBLISHING.md

```diff
@@ -37,14 +23,9 @@ Update Channels
 ---------------
 
 Etcher has a setting to include the unstable update channel. If this option is
-set, Etcher will consider both stable and unstable versions when showing the
-update notifier dialog. Unstable versions are the ones that contain a `beta`
-pre-release tag. For example:
-
-- Production unstable version: `1.4.0-beta.1`
-- Snapshot unstable version: `1.4.0-beta.1+7fde24a`
-- Production stable version: `1.4.0`
-- Snapshot stable version: `1.4.0+7fde24a`
```

Comment:
If we're still including the commit-hash on prerelease builds, perhaps part of this wording should be kept?